### PR TITLE
 [ADD] 12.0 : new module purchase_no_rfq that remove request for quotation concept in purchase workflow

### PR DESCRIPTION
**Pull request by legalsylvain**
This module extends the odoo purchase module, to remove 'Request for Quotation' state, simplifying
the workflow for the ends users.

Once installed :

* The menu item 'Purchase > Purchase > Requests for Quotation' is hidden. A single menu item 'Purchase Order'
  is available

* The states names of the purchase order is altered. 'RFQ' is replaced by 'Draft' and 'RFQ sent' by 'Sent'.

* The colors in the tree view is correctly set to ``decoration-info`` for 'draft' and 'sent' orders.

purchase_order_tree

* In the form view, all the RFQ names are removed or replaced by 'Purchase Order'.
  The module makes also the 'Print' button allways available and not only on 'draft' and 'sent' status.

* The option 'Print > Request For quotation' is also disabled.

purchase_order_form

CC : @quentinDupont (apps/deck/#/board/144/card/1424 : partie 1)

### CODE REVIEW

**Review by quentinDupont — APPROVED** [review]
Tested on clean Odoo and reviewed :heavy_check_mark: 

Really good for simplifying user experience

**Review by quentinDupont — CHANGES_REQUESTED** [review]
Just a minor change in translation https://github.com/legalsylvain/purchase-workflow/pull/1

**Review by quentinDupont — APPROVED** [review]

File: purchase_no_rfq/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+This module extends the odoo purchase module, to remove 'Request for Quotation' state, simplifying
+the workflow for the ends users.
```
Comment (hparfr) [inline purchase_no_rfq/readme/DESCRIPTION.rst]:
```suggestion
the workflow for the end users.
```

**Review by hparfr — APPROVED** [review]
Can't test on runbot.

Code read, looks ok.

**Review by bealdav — APPROVED** [review]

**Comment by legalsylvain** [comment]
hi @OCA/procurement-purchase-maintainers : some could merge this accepted PR ?

**Pull request merged**